lib/test/reporter/spec.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const {
+  ArrayPrototypePop,
+  ArrayPrototypeShift,
+  ArrayPrototypeUnshift,
+  SafeMap,
+  StringPrototypeRepeat,
+} = primordials;
+const assert = require('assert');
+const Transform = require('internal/streams/transform');
+const { inspectWithNoCustomRetry } = require('internal/errors');
+const { green, blue, red, white, gray } = require('internal/util/colors');
+
+
+const inspectOptions = { colors: true, breakLength: Infinity };
+
+const colors = {
+  '__proto__': null,
```

Comment:
```suggestion
  __proto__: null,
```
this should work the same since it's not computed, but there's no need to quote it

Reply:
Using a string literal is not the same thing as using computed prop name  (`LiteralPropertyName : StringLiteral` vs `ComputedPropertyName : [ AssignmentExpression ]`). I disagree with the above suggestion, I think it makes sense to keep all the keys quoted for consistency within the object. Anyway, no strong feelings so do however you like most.

Reply:
There's an eslint rule for the object key quoting style, whatever style the project selects should be enforced. Certainly "consistent, but unquote if possible" is a lintable style, but the ecosystem practice is pretty overwhelmingly "only quote when necessary, consistency be damned" :-)